girder/utility/webroot.py

```diff
@@ -57,6 +56,25 @@ def updateHtmlVars(self, vars):
         self.vars.update(vars)
         self.indexHtml = None
 
+    def addTemplateDirectory(self, templateDir):
+        """
+        Add a directory in which to search for templates.
+        """
+        self._templateLookup.directories.append(templateDir)
```

Comment:
Do you think it's safe to mutate the `TemplateLookup` instance variable `directories` directly like this? The constructor [does a little bit more work](https://github.com/zzzeek/mako/blob/master/mako/lookup.py#L186) (mostly calling [`posixpath.normpath`](https://docs.python.org/2/library/os.path.html#os.path.normpath)) before placing entries into `directories`.